src/components/structures/UserSettings.js

```diff
@@ -444,6 +444,27 @@ module.exports = React.createClass({
         );
     },
 
+    _renderReferral: function() {
+        const teamToken = window.localStorage.getItem('mx_team_token');
+        if (!teamToken) {
+            return null;
+        }
+        if (typeof teamToken !== 'string') {
+            console.warn('Team token not a string');
+            return null;
+        }
+        const href = window.location.origin +
+            `/#/register?referrer=${this._me}&team_token=${teamToken}`;
```

Comment:
Having spoken to @BenfLewis0, there's no reason why we wouldn't have referral links. Making this configurable would probably be desirable.